lisa/notifiers/log_file.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+from dataclasses import dataclass
+from typing import Any, List, Type, cast
+
+from dataclasses_json import dataclass_json
+
+from lisa import notifier, schema
+from lisa.environment import Environment
+from lisa.testsuite import TestResultMessage
+from lisa.util import constants, dump_file
+
+
+@dataclass_json()
+@dataclass
+class LogFileSchema(schema.TypedSchema):
```

Comment:
It looks no argument needed.

Reply:
@squirrelsc I made this class a subclass of the TypedSchema class because that's how it was for the corresponding schema for the Console notifier. Should I make this mirror the Schema in console.py or should I not make this class extend anything?

Reply:
This notifier doesn't need the runbook settings. So the whole class can be removed. In the notifier, `type_schema` can be overridden like,

```python
@classmethod
def type_schema(cls) -> Type[schema.TypedSchema]:
    return schema.Notifier
```